lib/src/external_module.cc

```diff
@@ -299,14 +296,14 @@ ActionOutcome ExternalModule::processRequestOutcome(const ActionRequest& request
         lth_jc::JsonContainer results { (out_txt.empty() ? "null" : out_txt) };
         return ActionOutcome { exit_code, err_txt, out_txt, results };
     } catch (lth_jc::data_parse_error& e) {
-        LOG_ERROR("'%1% %2%' output is not valid JSON: %3%",
-                  module_name, action_name, e.what());
-        std::string err_msg { "'" + module_name + " " + action_name + "' "
-                              "returned invalid JSON"  };
-        if (!err_txt.empty()) {
-            err_msg += " - stderr: " + err_txt;
-        }
-        throw Module::ProcessingError { err_msg };
+        LOG_DEBUG("Obtained invalid JSON on stdout for the %1%: %2%",
+                  request.prettyLabel(), e.what());
```

Comment:
Should we log the malformed stdout too?
We log only the ```lth_jc::data_parse_error``` exception's message, and then we include the stderr in the ```Module::ProcessingError``` exception we throw. But we don't log the offending stdout anywhere.